data/lib/workspace/generate/static/types.py

```
@@ -254,5 +255,15 @@ async def generate(data: GeneratorDatasource, collection):
 
         _apply_traits(pb, traits.get(type_id))
         collection.types[type_id].CopyFrom(pb)
+        if alpha_max_level is not None:
```

Ruff (0.15.12)

[error] Undefined name `alpha_max_level`

(F821)